src/sql/src/plan/with_options.rs

```diff
@@ -162,54 +162,51 @@ impl ImpliedValue for StringOrSecret {
     }
 }
 
-impl TryFromValue<Value> for Interval {
+impl TryFromValue<Value> for Duration {
```

Comment:
I'd add a note here that we use this to express notions of time because it's the more idiomatic type to use in Rust, e.g.

> We use `Duration` as the concrete type expressing spans of time because it is more idiomatic in Rust. However, we use `Interval` as the intermediate type because we have a robust set of APIs for parsing it as input from SQL.

or something like that.

Reply:
Good idea. Done.